test/unit/graphql/controller.test.ts

```diff
@@ -69,10 +69,43 @@ type Vote {
   decimal: Decimal
   big_decimal: BigDecimal
 }
-  `);
+  `;
+
+      const controller = new GqlEntityController(schema);
       const { builder } = await controller.createEntityStores(mockKnex);
 
-      expect(builder.toString()).toMatchSnapshot();
+      const createQuery = builder.toString();
+      expect(createQuery).toMatchSnapshot();
+    });
+
+    it('should work with autoincrement and nested objects', async () => {
```

Comment:
I feel like it could just be the only test case in this file (`it should work`).